src/edu/csus/ecs/pc2/clics/API202003/ClarificationService.java

```diff
@@ -21,11 +21,10 @@
 import edu.csus.ecs.pc2.core.model.Clarification;
 import edu.csus.ecs.pc2.core.model.ClarificationAnswer;
 import edu.csus.ecs.pc2.core.model.IInternalContest;
-import edu.csus.ecs.pc2.core.util.JSONTool;
 
 /**
  * WebService to handle languages
- * 
+ *
  * @author ICPC
```

Comment:
Hmm... obviously nothing to do with this PR, but "author ICPC" doesn't seem correct...

Reply:
I guess it has always been that way.  The stuff in the api202003 package was just moved from the pc2.services.web package.  The new stuff in the API202306 package, shouldn't have that, since it's all new code (pretty much).

Reply:
Yeah, I think there's a lot of "poorly" (or "never") reviewed stuff like this.   Doug begged for people to review his stuff, but when it didn't happen he eventually just threw up his hands and commited/merged it.   Can't say I blame him :)   I guess it's not really worth dedicating a lot of time toward fixing things like this in what is, essentially, old-and-destined-for-the-scrap-heap code.    If one of us DOES happen to find ourselves editing files with errors like this, I think it's worth "correcting"...